soconda.sh

```diff
@@ -56,14 +56,14 @@ done
 
 shift $((OPTIND-1))
 
-if [ "x${version}" = "x" ]; then
+if [ -z "${version}" ]; then
```

Comment:
Thanks, not sure what ancient shell I was using many years ago that did not like `-z` (old style habits die hard) but yes this is bash and that is a clearer expression.  I'll try to clean those up as I go :-)